src/PopChainManager/index.ts

```diff
@@ -8,6 +8,35 @@ export class PopChainManager {
     return closest(pop.targetEl, '[pop]', true);
   }
 
+  public getFullPopChain(pop: Pop, hideFullChain: boolean): Pop[] {
```

Comment:
Actually this is DFS. Anyways, does the order matter? Maybe not right now I assume.